cg/models/devices/illumina_run_directory_data.py

```diff
@@ -36,53 +36,53 @@
 }
 
 
-class FlowCellDirectoryData:
-    """Class to collect information about flow cell directories and their particular files."""
+class IlluminaRunDirectoryData:
```

Comment:
This is probably due to me being a bit fuzzy on what the intent is with this class. But is this not intended for Illumina flow cells? So would it not make sense to call the sequencing run for a flow cell here? 

For some cases, e.g. method names, I guess it would be to make the exposed methods have the same name? Although I see no inheritance from some interface. In for example comments, though, is there a benefit to calling it a sequencing run?